frontend/.env.example

```diff
@@ -10,6 +10,6 @@ NEXT_PUBLIC_SENTRY_DSN=
 NEXT_SERVER_CSRF_URL=http://backend:8000/csrf/
 NEXT_SERVER_DISABLE_SSR=false
 NEXT_SERVER_GRAPHQL_URL=http://backend:8000/graphql/
-NEXT_SERVER_GITHUB_CLIENT_ID=
+NEXT_PUBLIC_GITHUB_CLIENT_ID=
```

Comment:
Are we going to have this on the client (user browser)?

My understanding this convention works as `NEXT_SERVER_` -- for frontend server part and `NEXT_SERVER_` -- for frentend client part.